web/html/src/components/config-channels.tsx

```diff
@@ -214,15 +146,11 @@ class ConfigChannels extends React.Component<ConfigChannelsProps, ConfigChannels
 
   search() {
     if (this.state.filter === this.state.search.filter) {
-      this.setState({
-        view: "search",
-      });
       return Promise.resolve();
     } else {
       return Network.get(this.props.matchUrl(this.state.filter)).promise.then(data => {
         console.log(data);
```

Comment:
Not from this PR, but there are a bunch of console logs in this file that we probably don't want to ship.

Reply:
Done